src/lang/en-US.ts

```diff
@@ -926,6 +926,7 @@ const NewTransform: ResolvedIntlConfig['messages'] = {
 };
 
 const CustomRenderers: ResolvedIntlConfig['messages'] = {
+    'oauth.error.credentialsMissing': `need to complete OAuth`,
```

Comment:
Should this message be written in sentence case?

Reply:
No - it is matching these kind of errors

![image](https://user-images.githubusercontent.com/270078/232782947-938d7f78-f37e-49c4-b550-864f37de6feb.png)